Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
So the terminology may be confusing since a single point is usually called a feature, but in this case the number of `Cesium3DTileFeature`s that are created is not per-point but instead per batch, so there would be 8 in the `PointsWithBatchTableBinary` example, rather than 1000.

But yeah, if users decide to define batch table values per-point, this will definitely be a memory concern and we will need to use an idea like you suggested.